c/tskit/trees.h

```diff
@@ -265,15 +265,188 @@ typedef struct {
 @defgroup TREESEQ_API_GROUP Tree sequence API
 @{
 */
+
+/**
+@brief Initialises the tree sequence by (optionally) copying the supplied
```

Comment:
The brief is getting to much into the options weeds I think. How about "Initialises the tree sequence based on the specified table collection."

Reply:
Went with that and added detail in the main text.

Reply:
Doesn't look like you pushed?

Reply:
Whoops, done.